cip-xxxx/cip-xxxx.md

```diff
@@ -0,0 +1,159 @@
+To address this problem allow rewarding applications that do not
+involve Canton Coin, we propose introducing the ability for featured applications to create
+app activity markers without transfering Canton Coin. An app activity
+marker is equivalent to the existing app activity records created as
+part of a Canton Coin transfer with a fixed amount being determined by a
+new governance parameter.
+
+## Motivation
+
+Currently, as part of a Canton Coin transfer where the `provider`
+party is set to a featured application provider, an activity record is
+generated with the amount set to the amount of Canton Coin burned as
+part of that transfer plus the `extraFeaturedAppRewardAmount` defined
+in the `AmuletConfig` that the super validators voted on.
+
+This activity record can then be used to mint Canton Coin rewarding
+application providers for Canton Coin transfers generated as part of
+their application.
+
+While this works well to reward applications that inherently transfer
+Canton Coin, it does not reward applications that do not involve
+Canton Coin. It is possible to mitigate that by introducing artificial
+Canton Coin transfers into an application but that adds complexity
+during application development (even more so for transactions
+requiring external signing) and increases the size of the transaction
+and traffic costs without a clear benefit.
+
+Featured app activity markers address this by allowing applications to
+generate activity records and mint featured application rewards
+without the need to initiate Canton Coin Transfers.
+
+## Specification
+
+### Overview
+
+Featured applications get the ability to create
+`FeaturedAppActivityMarker` contracts, which have a constant value
+determined by the newly introduced `featuredAppActivityMarkerAmount`
+that is chosen through governance votes by the super validators (the
+default until changed is $1 USD). Tho governance process for changing
+this reuses the existing vote process to change `AmuletConfig` (also
+used for `extraFeaturedAppRewardAmount`) where one SV proposes a
+change and a 2/3 majority needs to accept.
+
+Automation run by the super validators converts these
+`FeaturedAppActivityMarker` contracts into the existing
+`AppRewardCoupon` contracts for the current open mining round with the
+US Dollar amount determined by `featuredAppActivityMarkerAmount` and converted
+to Canton Coin based on the Canton Coin conversion rate associated
+with that mining round.
+
+These `AppRewardCoupon` contracts can then be minted in the same
+fashion as activity records originating from Canton Coin transfers. In
+particular, they are minted from the existing minting pool/tranche for
+application rewards.
+
+One transaction can contain multiple markers for different providers
+just as one transaction can contain multiple Canton Coin
+transfers. This can be useful for composed applications where
+e.g. both an exchange and a registry application should get rewards
+for a DvP.
```

Comment:
```suggestion
transfers. This can be useful for composed applications like an
exchange where both an exchange and the asset registry applications should get rewards
for successfully settling a trade in a single, atomic DvP transaction.
```